packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/settings.dart

```diff
@@ -44,13 +45,20 @@ class Settings {
   /// and can be set to [Settings.CACHE_SIZE_UNLIMITED] to disable garbage collection.
   final int? cacheSizeBytes;
 
+  /// Whether to skip nested properties that are set to undefined during object serialization.
```

Comment:
```suggestion
  /// Whether to skip nested properties that are set to undefined during object serialization.
  ///
```

[Separate the first sentence of a doc comment into its own paragraph.](https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph)